jenkins/jmx-node-builds.yml

```diff
@@ -0,0 +1,23 @@
+type: jmx
+
+observation:
+  - name: jenkins-node-builds
+    metricNamespace: jenkins
+    objectName: io.jenkins:type=Node,name=Builds
+
+    metric:
+      - name: node.builds.count
```

Comment:
Use plural `node` -> `nodes`, few similar cases below too.

Reply:
Builds is in plural, do you think nodes should be also?

Reply:
Yeah.

Reply:
done